dictionaries/CallMap_80_delta.php

```diff
@@ -498,6 +498,10 @@
 'read_exif_data' => ['array', 'filename'=>'string', 'sections_needed='=>'string', 'sub_arrays='=>'bool', 'read_thumbnail='=>'bool'],
 'ReflectionClass::getConstants' => ['array<string,mixed>'],
 'ReflectionType::isBuiltin' => ['bool'],
```

Comment:
> **ReflectionType** has become abstract and **ReflectionType::isBuiltin()** has been moved to [ReflectionNamedType::isBuiltin()](https://www.php.net/manual/en/reflectionnamedtype.isbuiltin.php). 

(Source: https://www.php.net/manual/en/class.reflectiontype.php#reflectiontype.changelog)

So I think this line should be updated accordingly:

```suggestion
'ReflectionNamedType::isBuiltin' => ['bool'],
```

Reply:
I like this change but have 2 points:
 1. this one out of the scope of this PR (please understand me currently, [my first PR](#5204) is 5 month old, increasing a scope of this PR also increases a chance for this PR to be lost and outdated as the previous one)
 2. `ReflectionType::isBuiltin` is removed in PHP 8.0. As far as I understand, both `new` and `old` arrays doesn't provide instructions to psalm that the method is removed. `ReflectionNamedType::isBuiltin' => ['bool'],`  record is a valid one, but how to instruct psalm that `ReflectionType::isBuiltin` is removed?


PS: I also see 2 easy wins regarding changed files:
 1. `If they were just updated, the function/method will be present in the 'added' signatures.` uses `added` key, however we use `new`
 2. `dictionaries/CallMap_80_delta.php` doesn't contain indent and thus not really readable

But I've decided to limit a scope of this (and prev) PR

Reply:
Good points, in doubt I will open a PR separately.

I had a quick look at the [InternalCallMapHandler](https://github.com/vimeo/psalm/blob/6abce3525ac34f815aaa3b7953739b744ca5c39e/src/Psalm/Internal/Codebase/InternalCallMapHandler.php#L361-L402).

I think it should suffice to

1. not mention the removed method in the `dictionaries/CallMap.php`,
2. to add the removed method to the `old` key of the callmap corresponding to the version in which it was removed.

To determine the effective CallMap, Psalm goes backwards from the latest supported version by Psalm to the defined version. Psalm never goes the other way, so removed methods don't have to be declared removed explicitly.

Reply:
Sorry, never mind, the `isBuiltin` method is already declared correctly in the `CallMap_80_delta`:

![grafik](https://user-images.githubusercontent.com/495429/125342342-150f9480-e355-11eb-95d6-5f4ac9b7824d.png)